core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -97,6 +98,9 @@
   /** The journal system for writing journal entries and restoring master state. */
   protected final JournalSystem mJournalSystem;
 
+  /** The JVMMonitor Progress. */
+  private final JvmPauseMonitor mJvmPauseMonitor = new JvmPauseMonitor();
```

Comment:
can we put this feature under a boolean flag? so that we can enable it as needed.

Reply:
@ifcharming Sure!

Reply:
When `jvm.monitor.master.enable` is not true, do not instantiate this class.